sormas-app/app/src/main/java/de/symeda/sormas/app/component/controls/ControlSearchableSpinnerFieldListeners.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2018 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.symeda.sormas.app.component.controls;
+
+import android.view.View;
+import android.widget.AdapterView;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ControlSearchableSpinnerFieldListeners implements AdapterView.OnItemSelectedListener {
```

Comment:
Remove this class and use ControlSpinnerFieldListeners instead (which should work when ControlSearchableSpinnerField extends ControlSpinnerField)